parser/diff.go

```diff
@@ -104,6 +104,10 @@ func (p *DiffParser) Parse(r io.Reader) ([]*rdf.Diagnostic, error) {
 				}
 				prevState = diffLine.Type
 			}
+			if hunk.EOFNewline == diff.LineAdded {
+				state.newLines = append(state.newLines, "")
```

Comment:
Can you add a comment and a link to the PR comment so that future readers can understand it resolves the issue with GitHub suggestions.

Note that we don't confirm it works in the same way with other reporter such as GitLab, BitBucket etc...
Especially, I plan to add a feature that reviewdog could apply suggestions in local envorinment.
Then, this change won't work as expected for such other reporters. 
We should leave a comment about it.
Alternatively, it would be great if we can move the logic to GitHub review comment reporter.